### PR TITLE
onednn 1.5

### PR DESCRIPTION
**Pull request by alebcay**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
Previous repo URL redirects to new URL. Might be worth considering formula rename at some point? C header name remains unchanged at the moment.

### CODE REVIEW

**Comment by chenrui333** [comment]
maybe another alias? change looks good though.

File: Formula/mkl-dnn.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class MklDnn < Formula
   desc "Intel Math Kernel Library for Deep Neural Networks"
   homepage "https://01.org/mkl-dnn"
```
Comment (chenrui333) [inline Formula/mkl-dnn.rb:3]: the new homepage, `https://01.org/onednn`

**Comment by alebcay** [comment]
I think we should do a proper rename. The GitHub readme formally explains the name change.

**Comment by chenrui333** [comment]
Sounds good!

**Review by chenrui333 — COMMENTED** [review]
lgtm 💯

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**